orion-ui/src/maps/featureFlag.ts

```diff
@@ -0,0 +1,12 @@
+import { MapFunction } from '@/services/mapper'
+import { FlagResponse } from '@/types/flagResponse'
+import { FeatureFlag } from '@/utilities/permissions'
+
+export const mapFlagResponseToFeatureFlag: MapFunction<FlagResponse, FeatureFlag> = (source) => {
+  switch (source) {
+    // case 'magic':
+    //   return 'access:magic'
+    default:
+      throw new Error(`Invalid FlagResponse in mapper ${source}`)
```

Comment:
we cannot throw in a mapper like this. Otherwise if the backend adds a flag that we're not using it'll cause the app to fail at startup. I believe nebula-ui defaults to null